README.md

```diff
@@ -71,17 +83,27 @@ For example:
 ```
 
 ## Logging Programmatic Errors
-In order to log custom errors which will be visible in the platform, you can use `console.log("[LUMIGO_LOG] <YOUR_MESSAGE>");` from anywhere in your lambda code.
+With the tracer configured, simply call `console.log("[LUMIGO_LOG] <YOUR_MESSAGE>");` to cerate custom errors that are visible throughout the platform. This can be used anywhere in your Lambda code, and is included with the `@lumigo/tracer` package.
 
 ## Adding Execution Tags
-You can use `addExecutionTag` function to add an execution tag with a dynamic value.<br/>
-This value can be searched within the Lumigo platform.<br/>
-### Manual tracing
-Add `const lumigo = require('@lumigo/tracer')({ token: 'DEADBEEF' })`.<br/>
-Then use `lumigo.addExecutionTag('<key>', '<value>');` from anywhere in your lambda code.<br/>
-### Auto tracing
-Add `const lumigo = require('@lumigo/tracer');`.<br/>
-Then use `lumigo.addExecutionTag('<key>', '<value>');` from anywhere in your lambda code.<br/>
-### Limitation:
+You can add execution tags to a function with dynamic values using the parameter `addExecutionTag`.
+
+These tags will be searchable from within the Lumigo platform.
+
+### Adding tags for Manual tracing
+To add a tag to a manual trace statement:
+
+* Add `const lumigo = require('@lumigo/tracer')({ token: 'YOUR-TOKEN-HERE' })` to your code.
```

Comment:
Just like importing your tracer see the above example

Reply:
Can you clarify what you're trying to say here? Is it that I failed to include the API token, or is there something more fundamental that I've missed?

Reply:
No need to import the token. you can ignore my comment, it's just confusing